### PR TITLE
Allow password for docker registry to be inputted silently

### PR DESCRIPTION
**Pull request by 14rcole**
Previously, docker username and password had to be inputted through the terminal.  This allows the user to only give their username, then type their password at the prompt so that it is not displayed

### CODE REVIEW

**Comment by 14rcole** [comment]
@mrunalp @runcom PTAL

File: libkpod/common/common.go [hunk 1]
```diff
@@ -10,6 +10,11 @@ import (
 	"github.com/containers/image/types"
 )
 
+var (
+	// ErrNoPassword is returned if the user did not supply a password
+	ErrNoPassword = errors.New("password was not supplied")
```
Comment (rhatdan) [inline libkpod/common/common.go]: Can we make this a wrapped error
errors.Wrapf(syscall.EINVAL, password was not supplied")

Comment (14rcole) [inline libkpod/common/common.go]: Done

**Comment by rhatdan** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**